### PR TITLE
feat: support fax on message page

### PR DESCRIPTION
**Pull request by ele828**

### CODE REVIEW

File: src/components/ActionMenuList/index.js [hunk 1]
```diff
@@ -10,13 +10,15 @@ import DeleteMessageIcon from '../../assets/images/DeleteMessageIcon.svg';
...
+import PreviewIcon from '../../assets/images/Preview.svg';
+import DownloadIcon from '../../assets/images/Download.svg';
 
 import dynamicsFont from '../../assets/DynamicsFont/DynamicsFont.scss';
 
 import styles from './styles.scss';
 import i18n from './i18n';
 
-function ConfirmDeleteModal({
+export function ConfirmDeleteModal({
```
Comment (embbnux) [inline src/components/ActionMenuList/index.js]: Why export this function?

**Comment by RingCentralIntegrationDev** [comment]
Build successful! You can check your changes here: https://ec2-52-17-190-173.eu-west-1.compute.amazonaws.com/job/rc-js-widget-pr/56/

File: src/components/ActionMenuList/index.js (hunk 1 above)
Comment (ele828) [reply src/components/ActionMenuList/index.js]: used in tests, can be separated into independent files when needed

**Comment by RingCentralIntegrationDev** [comment]
Build successful! You can check your changes here: https://ec2-52-17-190-173.eu-west-1.compute.amazonaws.com/job/rc-js-widget-pr/60/

**Pull request merged**